src/brod.erl

```diff
@@ -179,6 +179,9 @@ start_client(BootstrapEndpoints, ClientId) ->
 %%       true | false | [{certfile, ...},{keyfile, ...},{cacertfile, ...}]
 %%       When true, brod will try to upgrade tcp connection to ssl using default
 %%       ssl options. List of ssl options implies ssl=true.
+%%     sasl (optional, default=undefined)
+%%       Credentials for SASL/Plain authentication.
+%%       {"PLAIN", "username", "password"}
```

Comment:
what would be the input look like for GSSAPI ?
in general, `{Tag, Data}` tuple is the most straightforward extendable schema.